tests/e2e/utils/helpers.go

```diff
@@ -257,6 +257,12 @@ func DeleteBins() error {
 	return nil
 }
 
+func DeleteCustomBinary(vmName string) {
+	vmPath := path.Join(GetBaseDir(), constants.VMDir, vmName)
+	// ignore error, file may not exist
+	os.RemoveAll(vmPath)
```

Comment:
Doesn't this result in a linter errror?

Reply:
nope! only if you assign the error to a variable I believe

Reply:
Oh interesting. Maybe just with my local linter settings.